RoddyCore/src/de/dkfz/roddy/config/Configuration.java

```diff
@@ -201,7 +201,10 @@ public String getName() {
      */
     @Override
     public String getID() {
-        return preloadedConfiguration.id;
+        if(preloadedConfiguration != null)
+            return preloadedConfiguration.id;
+        else
+            return "'Unnamed Configuration'";
```

Comment:
Why are there single ticks inside the double ticks?

Reply:
For readability.

Reply:
They don't belong there. You need to add them to the place, where the string is printed.